### ISSUE TITLE
Issue in Lumen 5.6

### ISSUE THREAD

**Issue by lucian-dragomir** [itsgoingd/clockwork#241]
Hi,

The extension seems not to work with **Lumen v5.6.*** - no information is returned into Clockwork Chrome Tab when calling application public root.

Steps i made: 
1. created new lumen app using: **lumen new testapp**
2. configured **bootstrap/app.php** file
     - uncomment line **$app->withFacades();**
     - uncomment line **$app->withEloquent();**
     - added **$app->register(Clockwork\Support\Lumen\ClockworkServiceProvider::class);** in the "Register Service Providers" section
3. renamed **.env.example** file to **.env** (default values used e.g. **APP_DEBUG=true**)
4. access application public root folder:
     http://..../testapp/public/
    the application responds with "**Lumen (5.6.1) (Laravel Components 5.6.*)**"
5. no infomation is returned in the Chrome-Clockwork plugin tab

Could you please provide configuration instructions in case something else is required or suggest a fix?

Thx.
Lucian

**Comment by itsgoingd** [comment]
Hey, please try following these debugging steps:

- check `storage/clockwork` directory, you should see a bunch of json files
- check your app http responses in the network dev tools tab, you should see bunch of `X-Clockwork-` headers
- try accessing `http://your-app/__clockwork/{id}` where id is the value of the `X-Clockwork-Id` header

**Comment by lucian-dragomir** [comment]
Hi,
Thank you for prompt response. 
I checked the application and i have no storage/clockwork folder and the http responses do not have the clockwork hreader (i created the clockwork under storage folder manually).
I executed a get to the /public route of the application and with XDebug activated I see that the service provider ClockworkServiceProvider->boot function is called on request.  Still i cannot figure why the http response does not contain the clockwork header.
If i display the link http://.../__clockwork/, the clockwork form is displayed empty ... witch is normal as i can't provide an clockwork id)
Thx,
Lucian

**Comment by itsgoingd** [comment]
There is actually an error in the documentation, it's missing a section that tells you to register the Clockwork middleware by updating your `app.php`:

```php
$app->middleware([
    Clockwork\Support\Lumen\ClockworkMiddleware::class
]);
```

I've also pushed an update that registers the middleware automatically, this will be released in Clockwork 2.2 later this week.

Sorry for the inconvenience.

**Comment by lucian-dragomir** [comment]
Hi @itsgoingd, thank you very much!
It works like a charm.
Lucian
